src/ClientInterface.php

```diff
@@ -26,192 +23,66 @@
-     *
-     * @return string|null
-     *
-     * @deprecated since version 2.0, to be removed in 3.0. Use getLastEvent() instead.
-     */
-    public function getLastEventId();
+    public function captureException(\Throwable $exception, ?Scope $scope = null): ?string;
 
     /**
      * Captures a new event using the provided data.
      *
-     * @param array $payload The data of the event being captured
-     *
-     * @return string
-     */
-    public function capture(array $payload);
-
-    /**
-     * Sends the given event to the Sentry server.
-     *
-     * @param Event $event The event to send
-     */
-    public function send(Event $event);
-
-    /**
-     * Translate a PHP Error constant into a Sentry log level group.
-     *
-     * @param int $severity PHP E_$x error constant
+     * @param array      $payload The data of the event being captured
+     * @param Scope|null $scope   An optional scope keeping the state
      *
-     * @return string Sentry log level group
+     * @return null|string
      */
-    public function translateSeverity(int $severity);
+    public function captureEvent(array $payload, ?Scope $scope = null): ?string;
 
     /**
-     * Provide a map of PHP Error constants to Sentry logging groups to use instead
-     * of the defaults in translateSeverity().
-     *
-     * @param string[] $map
-     */
-    public function registerSeverityMap($map);
-
-    /**
-     * Gets the user context.
-     *
-     * @return Context
-     */
-    public function getUserContext();
-
-    /**
-     * Gets the tags context.
-     *
-     * @return TagsContext
-     */
-    public function getTagsContext();
-
-    /**
-     * Gets the extra context.
-     *
-     * @return Context
-     */
-    public function getExtraContext();
-
-    /**
-     * Gets the runtime context.
-     *
-     * @return RuntimeContext
-     */
-    public function getRuntimeContext();
-
-    /**
-     * Gets the server OS context.
-     *
-     * @return ServerOsContext
-     */
-    public function getServerOsContext();
-
-    /**
-     * Sets whether all the objects should be serialized by the representation
-     * serializer.
-     *
-     * @param bool $value Whether the serialization of all objects is enabled or not
-     */
-    public function setAllObjectSerialize($value);
-
-    /**
-     * Gets the representation serialier.
+     * Logs the most recent error (obtained with {@link error_get_last}).
      *
-     * @return ReprSerializer
-     */
-    public function getRepresentationSerializer();
-
-    /**
-     * Sets the representation serializer.
+     * @param Scope|null $scope An optional scope keeping the state
      *
-     * @param ReprSerializer $representationSerializer The serializer instance
+     * @return null|string
      */
-    public function setRepresentationSerializer(ReprSerializer $representationSerializer);
+    public function captureLastError(?Scope $scope = null): ?string;
```

Comment:
I would move back the method to its original position for two reasons: it simpliefies the diff of this file (and so its review) and it stays with all the other `capture*` events before the generic one